addons/account_edi_ubl_cii_tax_extension/tests/test_ubl_cii_tax_extension.py

```diff
@@ -31,6 +31,6 @@ def test_classified_tax_category_codes(self):
             invoice.action_post()
             xml = self.env['account.edi.xml.ubl_bis3']._export_invoice(invoice)[0]
             root = etree.fromstring(xml)
-            for tax, node in zip(taxes, root.findall('.//{*}Item/{*}ClassifiedTaxCategory')):
+            for tax, node in zip(taxes, root.findall('.//{*}TaxTotal/{*}TaxSubtotal/{*}TaxCategory')):
```

Comment:
The test is named `test_classified_tax_category_codes` :thinking: 
Why this change?